dashboard/config/scripts_json/allthethings.script_json

```diff
@@ -5495,7 +5559,7 @@
     },
     {
       "seeding_key": {
-        "level.key": "2-3 Maze 6",
+        "level.key": "2-3 Maze 3",
```

Comment:
@code-dot-org/curriculum-platform - I added you to the review because of these changes. Not sure what's going on here. I didn't change this, but I'm guessing it was updated when I saved the allthethings script. Does it look ok?

Reply:
(also there are a few below and above that are similar)

Reply:
changes to this file look good! right now we generate changes to this file in preparation for moving to actually using this format, but currently this file is not used for anything. .script_json files are only used `is_migrated` scripts, which are currently just those in CSF / CSD / CSP 2021.